baseplate/observers/sentry.py

```diff
@@ -158,10 +163,19 @@ def __init__(self, hub: "Hub", client: raven.Client):
 
     def __call__(
         self,
-        context: RequestContext,
+        context: Any,
         exc_type: Optional[Type[BaseException]],
         value: Optional[BaseException],
         tb: Optional[TracebackType],
     ) -> None:
         self.raven.captureException((exc_type, value, tb))
-        self.original_print_exception(context, exc_type, value, tb)
+
+        if value and isinstance(value, ServerTimeout):
+            logging.warning(
```

Comment:
Should this be doing the `logger = logging.getLogger(__name__)` song and dance and using `logger`?